minify.js

```diff
@@ -260,7 +260,7 @@ module.exports = function express_minify(options) {
                 res.setHeader('Content-Type', 'text/css');
             } else if (coffee_match.test(contentType)) {
                 type = TYPE_COFFEE;
-                res.setHeader('Content-Type', 'text/javascript');
+                res.setHeader('Content-Type', 'application/javascript');
```

Comment:
See also:
* http://www.w3.org/TR/html401/interact/scripts.html#adef-type-SCRIPT

So **it does** knock out some browser compatibility... there are hundreds of articles on why use one or the other but the maximum compatibility with a project can be based for all browsers or a select few... depends on what @SummerWish wants to do ... support only the recent standards and a smaller set of applications or as much as possible. This can be loosely related to platform support as well between Intel Architecture and SunOS support as well.